### ISSUE TITLE
Discussion: Safety

### ISSUE THREAD

**Issue by trosel** [ArcadeCity/bullrun#1]
Reputation can go a long way in terms of providing safety and ease of mind for participants, but there needs to be additional ways to improve safety to “bootstrap” comfortable usage. 

As a driver/rider, I want to know that I am in a car with someone with a real identity, so that if something goes wrong, that identity can be used to help solve problems. 

As a rider, I want to be able to share my trip status with a friend, so that they can assist if anything goes wrong. 

## Vetting

Drivers and Riders should be able to pay a third party security firm to “vet” them, then have this proof shown in the app. 

i’m thinking maybe a ZK Proof or a colored coin (RGB protocol) that is optionally displayed in the app. 

multiple firms can compete for the best vetting and rates. 

If someone is vetted in the app, they will probably get easier rides or more riders. 

## Reporting Problems

If something happens on the trip (god forbid, an assault or something), there should be a way to share real rider or driver details with law enforcement. 

## Sharing trip status

this is a simple one. I want to share the status of my trip with a friend. if i go missing (don’t confirm my arrival?) then my friend is notified. Uber and Lyft both have this. 

—-

I think underlying all of this is the idea that a participants real name and details exist externally from the platform and are shareable in extreme situations. Not sure if feasible, but much to discuss. 

Thoughts?

**Comment by supertestnet** [comment]
All of those are good ideas. The part about sharing trip status, in particular, is easy to implement and helps a lot with ease of mind. Regarding third party security firms, this reminds me of the badges that are under discussion in the nostr telegram. A nip is being worked on that allows third parties to assign verification badges to people that nostr clients can display as green checkmarks or similar. We could use the same principle to make arcade city display a "vetted" badge if a driver KYC's with a third party. I think Christopher also has ideas about using group-vetting with a concept he invented called guilds. He can speak more to that.

Regarding assault reporting, one thing I don't personally want is for any driver to feel forced to give away his real identity in order to drive people around for money. I have family members who struggle to find work because they have a criminal record. I've had experience with 15 year olds who can't find work because they don't have identity documents. I want them all to feel unburdened -- you can always be a bullrun driver (or eventually do bicycle delivery) by just opening up an app. I am fine with client software such as arcade city having a filter with allows users to filter out un-KYC'd drivers, if they want. My hope is that not everyone uses that filter so that people in financial straits with no identity documents can still make money doing bullruns.

**Comment by trosel** [comment]
> Regarding assault reporting, one thing I don't personally want is for any driver to feel forced to give away his real identity in order to drive people around for money. I have family members who struggle to find work because they have a criminal record

Yes, discriminating based on past legal issues would be a problem for people being vetted to certain high degrees. 

Maybe the middle ground is a simple KYC verification that (if something happens) is shared with authorities. Almost like a dead man trigger or something? Just brainstorming. 

Nice thing about uber is that bad things don’t usually happen because you literally can’t get away with it.
